ide/netbeans/pom.xml

```diff
@@ -124,7 +124,7 @@ under the License.
 
   <build>
     <plugins>
-      <plugin>
+    <plugin>
```

Comment:
yes ok I won't sorry for causing the trouble should I fix it and open the pr again  .

Reply:
No need to open a new PR, just revert this sole change and make a new commit: it will be added to this PR.
Once the review is finished, you might be asked to squash all the commits, but for the moment it will work like this.

Reply:
ok